app/frontend/components/pages/sendAda/confirmTransactionDialog.tsx

```diff
@@ -76,8 +76,7 @@ class ConfirmTransactionDialogClass extends Component<Props, {}> {
                 <div className="review-amount">{pool.ticker}</div>
                 <div className="review-label">Tax</div>
                 <div className="review-amount">
-                  {pool.rewards &&
-                    (pool.rewards.ratio[0] * 100) / pool.rewards.ratio[1]}%
+                  {pool.rewards && (pool.rewards.ratio[0] * 100) / pool.rewards.ratio[1]}%
```

Comment:
if I understand right, this division may produce nasty values, like "66.66666667", shouldn't we wrap some rounding around it?

Reply:
Yeah, I think we should as well, I created a helper for this. It is used only in this one place but I thought we should separate its logic from `confirmTransactionDialog`. Open to suggestions tho, creating a helper for this might not seem necessary.